2020/dec12/ship/part2/runner.go

```diff
@@ -0,0 +1,84 @@
+package part2
+
+import (
+	"fmt"
+
+	"github.com/ablqk/adventofcode/2020/dec12/ship"
+)
+
+func New(code []ship.Instruction, waypoinyLat, waypointLong int) ship.Runner {
+	return &runner{
+		code:     code,
+		waypoint: coordinate{long: waypointLong, lat: waypoinyLat},
+	}
+}
+
+type runner struct {
+	code     []ship.Instruction
+	ship     coordinate // 0,0 is the ship's starting position
+	waypoint coordinate // 0,0 is the ship's current position
```

Comment:
is this comment really accurate?

Reply:
`waypoint coordinate // this is a vector, so 0,0 is the ship's current position`
Better?

Reply:
`this is a vector of the ship's bearing` would be enough